packages/financial-templates-lib/src/price-feed/TraderMadePriceFeed.js

```diff
@@ -70,9 +70,20 @@ class TraderMadePriceFeed extends PriceFeedInterface {
 
     // Set first price time in `historicalPrices` to first non-null price.
     let firstPriceTime;
-    for (let p in this.historicalPricesMinute) {
-      if (this.historicalPricesMinute[p] && this.historicalPricesMinute[p].openTime) {
-        firstPriceTime = this.historicalPricesMinute[p];
+
+    // If no minute-interval prices are available, try the hourly-interval:
+    // Note: The TraderMade API does not update /timeseries data over the weekend,
+    // so to handle this case we can fall back on the longer lookback window of the
+    // hourly timeseries. (The lookback limit for the minute inverval is 2 days, while the limit
+    // for the hourly interval is 2 months). This fall back logic will only work
+    // if `this.hourlyLookback` is configured long enough such that there is an hourly price
+    // available.
```

Comment:
again nit about line length
```suggestion
// Note: The TraderMade API does not update /timeseries data over the weekend, so to handle this case we can fall back
// on the longer lookback window of the hourly timeseries. (The lookback limit for the minute inverval is 2 days, while
// the limit for the hourly interval is 2 months). This fall back logic will only work if `this.hourlyLookback` is
// configured long enough such that there is an hourly price available.
```

BTW you can set VS code to show you vertical lines at our line length. add this to your vs code config:
```
"editor.rulers": [
        80,
        120
    ],
```